test/malli/instrument_test.cljs

```diff
@@ -196,3 +196,22 @@
 (deftest ^:simple check-test
   (let [results (mi/check)]
     (is (map? results))))
+
+(deftest instrument-external-test
```

Comment:
you'll need to mark this test with `^:simple` to not run during advanced compilation on cljs.

Reply:
Interesting, did instrumentation not work with advanced compilation?

Reply:
Not this version (as opposed to the `malli.instrument.cljs` ns which uses macros) because it is implemented to replace the JS functions at runtime by setting their value by name lookup. After advanced compilation the names will all be minified and there is no way (that I know of) to access the minified name.